Packs/FeedMicrosoftIntune/Integrations/FeedMicrosoftIntune/FeedMicrosoftIntune_description.md

```diff
@@ -0,0 +1,2 @@
+## Microsoft Intune URL's web scraper:
+https://docs.microsoft.com/en-us/intune/fundamentals/intune-endpoints
```

Comment:
I remember in the original PR you opened you had a very nice detailed description about the use-case and whatnot - can we add that here?